velox/expression/CastExpr.cpp

```diff
@@ -123,6 +123,37 @@ void applyIntToDecimalCastKernel(
   });
 }
 
+template <typename TInput>
+VectorPtr applyDecimalToIntCast(
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context,
+    const TypePtr& toType) {
+  VectorPtr result;
+  context.ensureWritable(rows, INTEGER(), result);
```

Comment:
What if toType is some other kind of integer, SMALLINT or TINYINT?

Reply:
to unblock internal tests only supports int32.

Reply:
Would you add support for all integers for completeness?

Reply:
On it.